pyanaconda/modules/storage/devicetree/model.py

```diff
@@ -285,6 +285,7 @@ def _mark_protected_devices(self):
         identify protected devices.
         """
         protected = []
+        protected_w_ancestors = []
```

Comment:
I wonder what `w` means here. Is it `with` or `without`? I would like to not use this abbreviations.

Reply:
Updated